glitter_events/models.py

```diff
@@ -33,11 +33,29 @@ def get_absolute_url(self):
 
 
 @python_2_unicode_compatible
+class Location(models.Model):
+    title = models.CharField(max_length=32, db_index=True)
+    slug = models.SlugField(max_length=32, unique=True)
+    location = models.CharField(max_length=100, unique=True)
+
+    class Meta:
+        ordering = ('title',)
+
+    def __str__(self):
+        return self.title
+
+    def get_absolute_url(self):
+        return reverse('glitter-events:location-event-list', kwargs={
```

Comment:
Downside of the feature flag in `urls.py` - if it's disabled then this will cause an exception.

Add a fallback? Could just go to the events homepage.